Adding tutorial links to documentation

hb326 opened the pull request:
With this pull request some links to the ESMValTool tutorial are added to the documentation.

Closes #1926 


* * *

Automated checks pass, status can be seen below the pull request:

- [x] Circle/CI tests pass. If the tests are failing, click the `Details` link to find out why.
- [ ] Preferably Codacy code quality checks pass, however a few remaining hard to solve Codacy issues are still acceptable. If there is an error, click the link to find out why. If you suspect Codacy may be wrong, please ask by commenting.
- [x] The documentation is building successfully on readthedocs and looks well formatted, click the `Details` link to see it.


* * *

If you need help with any of the items on the checklists above, please do not hesitate to ask by commenting in the issue or pull request.

SarahAlidoost commented on doc/sphinx/source/develop/recipe.rst:
```diff
@@ -15,3 +15,8 @@ the specific parameters needed by a recipe that runs a personal diagnostic are:
 
 i.e. the full path to the personal diagnostic that the user needs to run.
+
+There is also an episode available in the 
```
```suggestion
There is also a lesson available in the 
```

SarahAlidoost commented on doc/sphinx/source/quickstart/configuration.rst:
```diff
@@ -21,3 +21,8 @@ To install the default configuration file in the default location, run
 the ``esmvaltool`` command can find the data on your system, before it can run
 a recipe.
+
+There is an episode available in the 
```
```suggestion
There is a lesson available in the 
```

SarahAlidoost reviewed (changes requested):
@hb326 thanks a lot. I suggested replacing "episode" with "lesson". I think the former is a technical word, whereas the latter is more user-friendly. Also, could you please add a similar change to the part of the documentation which is about the [conda installation](https://docs.esmvaltool.org/en/latest/quickstart/installation.html#conda-installation)?  The related lesson can be found [here](https://esmvalgroup.github.io/ESMValTool_Tutorial/02-installation/index.html). Thank you.

hb326 commented:
Thanks for the suggestions, @SarahAlidoost! 
I have changed the wording in the two cases you suggested, and I also added the link to the Tutorial for the installation part of the documentation.

SarahAlidoost reviewed (approved):
@hb326 thank you for addressing the comments. I approved the changes.

hb326 commented:
Hey @axel-lauer, can you merge this? Not sure if I could/should do it myself. Thanks!
